routers/user_router.py

```diff
@@ -58,4 +58,25 @@ def delete_user(user_email: str, session: Session = Depends(get_db)) -> user_sch
         service.remove_user(user_email)
     except ValueError as e:
         raise HTTPException(status_code=404, detail=str(e))
-    return user_schema.UserDeletedResponse(message=f"user with email{user_email}, was successfully deleted")
\ No newline at end of file
+    return user_schema.UserDeletedResponse(message=f"user with email{user_email}, was successfully deleted")
+  
+@router.put("/users/{user_email}/profile-picture", status_code=200,response_model=user_schema.UserUpdate)  
+def Update_profile_picture(user_email: str, image: UploadFile, session: Session = Depends(get_db)) -> user_schema.UserUpdate:
+    service = UserService(session=session)
+    try:
+        image_path = service.generate_profile_picture_path(user_email, image)
```

Comment:
I think at this level, you should Add validation to ensure the uploaded file is of an acceptable type and size before saving it. This reduces the risk of storing overly large files.

Reply:
Large files should not be a problem but we can definitely try to have a limit for files